stripe/multipart_data_generator.py

```diff
@@ -18,8 +18,11 @@ def add_params(self, params):
             self._write(self.param_header())
             self._write(self.line_break)
             if hasattr(value, 'read'):
-                self._write("Content-Disposition: form-data; name=\"%s\"; "
-                            "filename=\"%s\"" % (key, value.name))
+                self._write("Content-Disposition: form-data; name=\"")
+                self._write(key)
+                self._write("\"; filename=\"")
+                self._write(value.name)
+                self._write("\"")
```

Comment:
I changed the code to not rely on Python's string formatting to handle the (admittedly improbable) case where the key is a byte array.